terraso_backend/apps/core/models/landscapes.py

```diff
@@ -150,6 +150,10 @@ def get_default_group(self):
 
         return landscape_group.group
 
+    # used for sitemap generation
+    def get_absolute_url(self):
```

Comment:
Question: Where is this function being used? I don't see this referenced in this PR

Reply:
It is used by the built-in [Django sitemap component](https://docs.djangoproject.com/en/4.2/ref/contrib/sitemaps/). It is called over the iterable (Landscape.objects.all()).